ViewArticle.jsp

```diff
@@ -236,7 +236,7 @@ footer
 <a href="Controller?page=home">Home</a>
 <a href="Controller?page=aboutus">About us</a>
 <a href="Controller?page=contact">Contact</a>
-<a href="Controller?page=admin">Admin</a>
+<a href="Controller?page=admin">Author</a>

```

Comment:
@mjhmohamed 

Why `Author `is more appropriate than `Admin`?

Hui

Reply:
I don't have an answer.

As i said earlier, I have difficulties communicating with our programmer, i did ask him to reply to your reviews a week ago, still he hasn't replied to me since then.

IBRAHIM.